providers/openlineage/provider.yaml

```
@@ -159,6 +159,9 @@ config:
       execution_timeout:
         description: |
           Maximum amount of time (in seconds) that OpenLineage can spend executing metadata extraction.
+          Note that other configurations, sometimes with higher priority, such as core.task_success_overtime,
+          may also affect how much time OpenLineage has for execution. For more details, see:
+          https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#task-success-overtime
```

```suggestion
          Note that other configurations, sometimes with higher priority, such as
          `[core] task_success_overtime <https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#task-success-overtime>`_,
          may also affect how much time OpenLineage has for execution.
```

Let's link it instead? Something like this.

---

Done, also adjusted all other url's in the same way.